zh/deploy-on-gcp-gke.md

```diff
@@ -8,728 +8,346 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/deploy-on-gcp-gke/']
-      tidb_operator_id           = module.tidb-operator.tidb_operator_id
-      gcp_project                = var.GCP_PROJECT
-      gke_cluster_location       = local.location
-      gke_cluster_name           = ${gke_cluster_name}
-      cluster_name               = "example-tidb-cluster"
-      cluster_version            = "v4.0.4"
-      kubeconfig_path            = local.kubeconfig
-      tidb_cluster_chart_version = "v1.0.0"
-      pd_instance_type           = "n1-standard-1"
-      tikv_instance_type         = "n1-standard-4"
-      tidb_instance_type         = "n1-standard-2"
-      monitor_instance_type      = "n1-standard-1"
-      pd_node_count              = 1
-      tikv_node_count            = 2
-      tidb_node_count            = 1
-      monitor_node_count         = 1
-    }
+    NAME                              READY   STATUS    RESTARTS   AGE
+    tidb-discovery-5cb8474d89-n8cxk   1/1     Running   0          47h
+    tidb-monitor-6fbcc68669-dsjlc     3/3     Running   0          47h
+    tidb-pd-0                         1/1     Running   0          47h
+    tidb-pd-1                         1/1     Running   0          46h
+    tidb-pd-2                         1/1     Running   0          46h
+    tidb-tidb-0                       2/2     Running   0          47h
+    tidb-tidb-1                       2/2     Running   0          46h
+    tidb-tikv-0                       1/1     Running   0          47h
+    tidb-tikv-1                       1/1     Running   0          47h
+    tidb-tikv-2                       1/1     Running   0          47h
     ```
 
-    > **注意：**
-    >
-    > - 每个集群的 `cluster_name` 必须是唯一的。
-    > - 为任一组件实际创建的总节点数 = 配置文件中的节点数 * 配置的可用区的个数（Regional 集群默认为 3）。
+## 访问数据库
 
-2. 修改完成后，执行以下命令来创建集群。
+### 准备一台可以访问集群的机器
 
-    {{< copyable "shell-regular" >}}
-
-    ```bash
-    terraform init
-    terraform apply
-    ```
-
-## 扩容
-
-如果需要扩容 TiDB 集群，可执行以下步骤：
-
-1. 增大 `.tfvars` 文件中 `pd_count`、`tikv_count`、`tidb_count` 变量。
-2. 运行 `terraform apply`。
-
-> **警告：**
->
-> 由于缩容过程中无法确定哪个节点会被删除，因此目前不支持集群缩容。通过修改 `tikv_count` 来进行缩容可能会导致数据丢失。
-
-扩容过程会持续几分钟，你可以通过以下命令来持续观察进度：
+我们为 TiDB 集群创建的是内网 LoadBalancer。我们可在集群 VPC 内创建一台[堡垒机](https://cloud.google.com/solutions/connecting-securely#bastion)访问数据库。
 
 {{< copyable "shell-regular" >}}
 
-```
-kubectl --kubeconfig credentials/kubeconfig_${gke_cluster_name} get po -n ${tidb_cluster_name} --watch
-```
-
-例如，可以将 `tidb_count` 从 1 改为 2 来扩容 TiDB：
-
-```hcl
-tidb_count = 2
+```shell
+gcloud compute instances create bastion \
+  --machine-type=n1-standard-4 \
+  --image-project=centos-cloud \
+  --image-family=centos-7 \
+  --zone=<your-region>-a
 ```
 
 > **注意：**
 >
-> 增加节点数量会在每个可用区都增加节点。
+> `<your-region>-a` 为集群所在的区域的 a 可用区，比如 us-central1-a 。也可在同区域下的其他可用区创建堡垒机。
 
-## 自定义
+### 安装 MySQL 客户端并连接
 
-你可以更改 `variables.tf` 中的默认值，例如集群名称和镜像版本等，但更建议在 `terraform.tfvars` 文件或其它相关文件中来指定值。
+待创建好堡垒机后，我们可以通过 SSH 远程连接到堡垒机，再通过 MySQL 客户端 来访问 TiDB 集群。
 
-### 自定义 GCP 资源
+用 SSH 连接到堡垒机：
```

Comment:
Suggest using `1. 2. 3...` for these consecutive steps:
```
1. 用 SSH 连接到堡垒机：
2. 安装 MySQL 客户端：
3. 连接到 TiDB 集群：
```

Reply:
done